src/app/project/description/description.component.ts

```diff
@@ -0,0 +1,276 @@
+import { ENTER, COMMA } from '@angular/cdk/keycodes';
+import { Component, Inject, OnInit } from '@angular/core';
+import { FormBuilder, FormControl, FormGroup, Validators } from '@angular/forms';
+import { MatChipInputEvent } from '@angular/material/chips';
+import { ActivatedRoute, Params, Router } from '@angular/router';
+import {
+    ApiResponseData,
+    ApiResponseError,
+    KnoraApiConnection,
+    ProjectResponse,
+    ReadProject,
+    StringLiteral,
+    UpdateProjectRequest
+} from '@dasch-swiss/dsp-js';
+import { DspApiConnectionToken } from 'src/app/main/declarations/dsp-api-tokens';
+import { ErrorHandlerService } from 'src/app/main/services/error-handler.service';
+import { NotificationService } from 'src/app/main/services/notification.service';
+import { Session, SessionService } from 'src/app/main/services/session.service';
+import { ProjectService } from 'src/app/workspace/resource/services/project.service';
+import { CacheService } from '../../main/cache/cache.service';
+
+@Component({
+    selector: 'app-description',
+    templateUrl: './description.component.html',
+    styleUrls: ['./description.component.scss']
+})
+
+export class DescriptionComponent implements OnInit {
+
+    // loading for progress indicator
+    loading: boolean;
+
+    // permissions of logged-in user
+    session: Session;
+    sysAdmin = false;
+    projectAdmin = false;
+
+    // project uuid; as identifier in project cache service
+    projectUuid: string;
+
+    // project data
+    project: ReadProject;
+
+    color = 'primary';
```

Comment:
This member is not used in the file or template

Reply:
done in [ab03e17](https://github.com/dasch-swiss/dsp-app/pull/928/commits/ab03e177a0cf0cce96f985c7abb728fd33929a5c)